flytekit/clis/sdk_in_container/run.py

```diff
@@ -422,9 +423,9 @@ def dump_flyte_remote_snippet(execution: FlyteWorkflowExecution, project: str, d
     )
 
 
-def get_workflows_in_file(filename: str) -> typing.List[str]:
+def get_entities_in_file(filename: str) -> typing.Tuple[typing.List[str], typing.List[str]]:
```

Comment:
Can we not return a single list of workflows and tasks cause we're anyways combining them in `list_commands` as follows:

```python
workflows, tasks = get_entities_in_file(self._filename)
workflows.extend(tasks)
```

Reply:
Or better, return a dictionary of lists to keep it clean, say: `{"workflows": [...],  "tasks": [...]}`